frontend/src/bundles/friends/components/friend-details/friend-details.tsx

```diff
@@ -93,9 +93,12 @@ const FriendDetails = ({
             </div>
 
             <div className="mb-4 flex w-full items-center justify-between">
-                <h3 className="text-lm-grey-200 text-xl font-extrabold leading-6">
-                    Achievements
-                </h3>
+                {achievements?.length > 0 && (
```

Comment:
I think it should be fixed on backend, seems like we are returning either error or null if user don't have achievements. should return empty array.

Reply:
fixed